src/ApparentHorizons/StrahlkorperGr.cpp

```diff
@@ -360,56 +361,93 @@ double get_spin_magnitude(const std::array<DataVector, 3>& potentials,
 namespace StrahlkorperGr {
 
 template <typename Frame>
-tnsr::i<DataVector, 3, Frame> unit_normal_one_form(
+void unit_normal_one_form(
+    const gsl::not_null<tnsr::i<DataVector, 3, Frame>*> result,
     const tnsr::i<DataVector, 3, Frame>& normal_one_form,
     const DataVector& one_over_one_form_magnitude) noexcept {
-  auto unit_normal_one_form = normal_one_form;
+  destructive_resize_components(result, normal_one_form.begin()->size());
```

Comment:
should not need to do the destructive_resize_components when setting `*result =` an existing object as operator= should do any resize it needs, so omit it here